pupil_src/capture/world.py

```diff
@@ -149,9 +149,14 @@ def get_timestamp():
     default_plugins = [('Log_Display',{}),('Dummy_Gaze_Mapper',{}),('Display_Recent_Gaze',{}), ('Screen_Marker_Calibration',{}),('Recorder',{}),('Pupil_Remote',{})]
 
 
+    def should_call_ui_functions(window):
```

Comment:
We should do that anyway, because if we minimise the main window it would set `g_pool.iconified = True` which affects `eye` behaviour. This should not be the case.